CHANGELOG.md

```diff
@@ -41,6 +41,9 @@
 - Removed `driConfigureSupervisorCluster.ps1` from the \SampleScripts\ directory as functionality now provided using the `Invoke-DriDeployment` cmdlet.
 - Removed `driDeployTanzuCluster.ps1` from the \SampleScripts\ directory as functionality now provided using the `Invoke-DriDeployment` cmdlet.
 - Removed `driUndoDeployment.ps1` from the \SampleScripts\ directory as functionality now provided using the `Invoke-UndoDriDeployment` cmdlet.
+- Added `Invoke-GeneratePrivateKeyAndCsrFiles` cmdlet to generate private key and CSR files using Openssl, common name and input parameters.
+- Added `Invoke-RequestSignedCertificateByMsca` cmdlet to connects to MSCA web enrollment to sign a CSR.
```

Comment:
Why not `Invoke-RequestSignedCertificate` and have a -certificateAuthority switch to do MSCA or OpenSSL?

Reply:
signing off openSSL is not part of certGenVVS capability.  I have modified the function to take in a switch value between msca and openssl.  and implement openssl signing as part of the Invoke-RequestSignedCertificate function.  
addressed in commit 9d50a853ffce55094985dea3d9e658670a7ab67a